README.md

```diff
@@ -79,53 +115,90 @@ Create a `config.yml` file and copy the `default-config.yml` content to it and c
 
 ```
 npm run deploy
 ```
+
 This will:
-- Create APIKey in SecretsManager in us-east-1 and its replicas in all enabled regions, this key will be used to secure the events that are sent from various regions to the Cloud Responder (installed in us-east-1)
+
+- Create APIKey in SecretsManager in us-east-1 and its replicas in all enabled regions, this key will be used to secure the events that are sent from various regions to _Responder_ (installed in us-east-1)
 - Deploy the Serverless Cloud Responder application in us-east-1
 - Deploy this EventBridge [CloudFormation template](aws-eventbridge.yml) into every region
 
-### Remove
+<p>&nbsp;</p>
+
+# Remove
 
 ```
 npm run remove
 ```
 
 This will:
+
 - Remove the EventBridge CloudFormation stacks in all regions
 - Remove the Serverless app
 - Remove the APIKey from SecretsManager in every region
+<p>&nbsp;</p>
 
-## How it works
+## How it works:
+
+<p>&nbsp;</p>
 
 ### On AWS
 
-*The EventBridge Rules*<br />
+_The EventBridge Rules_<br />
 On AWS, the installation creates a Rule on EventBridge. This rule capture CloudTrail events and send these events to a Target.
 The Target is an API Connection that sends the event to the Cloud Responder via a HTTP request.
 Because the EventBridge is region specific, it has to be done in every region.
 
-*The Serverless app*<br />
+_The Serverless app_<br />
 The serverless app exposes a HTTP endpoint that will receive POST requests sent by the API Connections installed in all regions. The the KeyPOST requests are authenticated via an APIKey tha is shared between the API Connection and the Serverless HTTP endpoint (API Gateway). An Authorizer (Lambda function) will authenticate the request by checking the API Key. Once authenticated the detection/capture sequence can start.
 
 ### Anywhere
 
-*Detection/Capture sequence*<br />
+_Detection/Capture sequence_<br />
 The capture sequence, is a rule that is going to be run again the incoming event. The rule checks if the event is of interest. The rule must return an array of findings if the the event is of interest. Returning one or more findings will start a flow sequence.
 
-*The flow sequence*<br />
+_The flow sequence_<br />
 A flow sequence will start when a finding has been discoverd, it a series of actions that can be taken sequentially or in parallel.
 
+<p>&nbsp;</p>
+
 # Documentation
 
-[General documentation](./documentation)
+[General documentation](./documentation/README.md#table-of-contents)
+
+  <p>&nbsp;</p>
 
 # Out-of-the-box Actions
 
-- [Send Slack](actions/SendSlack/documentation.md)
-- [Send Email](actions/SendEmail/documentation.md)
-- [Wait](actions/Wait/documentation.md)
\ No newline at end of file
+- [Send Slack](actions/SendSlack/documentation.md#send-slack-action)
+- [Send Email](#out-of-the-box-actions)
+- [Send Discord](actions/SendDiscord/documentation.md#send-discord-action)
+- [Wait](#out-of-the-box-actions)
```

Comment:
@dannyb1983  can you add a link to the Google Chat action documentation here?

Reply:
Goole chat is added 👍